sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -781,3 +782,68 @@ case class SchemaOfJson(
 
   override def prettyName: String = "schema_of_json"
 }
+
+/**
+ * A function that returns the number of elements in outer JSON Array.
+ */
+@ExpressionDescription(
+  usage = "_FUNC_(jsonArray) - Returns the number of elements in outer JSON Array.",
+  arguments = """
+    Arguments:
+      * jsonArray - A JSON array is required as argument. An Exception is thrown if any
+          other valid JSON strings are passed. `NULL` is returned in case of invalid JSON.
```

Comment:
if any other valid JSON strings are passed, we should return null as well?

Reply:
Won't it be good if we inform users that this function excepts JSON Array only?

Reply:
then shall we inform users about invalid JSON as well?

Reply:
Ideally we should. I just returned `NULL` because other JSON functions returns `NULL` as well.